ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -105,9 +113,7 @@ def main(argv=sys.argv[1:]):
         return 1
 
     # output errors
-    report = {}
-    for filename in files:
-        report[filename] = []
+    report = defaultdict(list)
```

Comment:
This change broke the report since only files with error appear in the report now. See #134 for reverting this part.